openedx/features/ucsd_features/models.py

```diff
@@ -0,0 +1,64 @@
+from django.conf import settings
+from django.db import models
+
+
+# Backwards compatible settings.AUTH_USER_MODEL
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+
+class AdditionalRegistrationFields(models.Model):
+    """
+    This model contains two extra fields that will be saved when a user registers.
+    The form that wraps this model is in the forms.py file.
+    """
+    user = models.OneToOneField(USER_MODEL, null=True)
```

Comment:
Are we sure that we need null to be true here? Secondly, If we have to allow null than I think that should only be the scenario where on_delete=models.SET_NULL.

Reply:
Team UCSD code :D

Reply:
ok, but keep in mind that git blame doesn't know that.

Reply:
I've added `on_delete=models.SET_NULL` constraint though.